src/components/ArticleList.jsx

```diff
@@ -0,0 +1,28 @@
+import { useEffect, useState } from "react";
+import { getArticles } from "../api";
+import ArticleCard from "./ArticleCard";
+
+export default function ArticlesList() {
+    const [articles, setArticles] = useState([]);
+  
+    useEffect(() => {
+      getArticles()
+        .then((data) => {
+          setArticles(data);
+        })
```

Comment:
What might be a more descriptive name that data as we should know what this promise actually resolves with